1.9/usage/marathon/troubleshooting.md

```diff
@@ -0,0 +1,18 @@
+---
+post_title: Troubleshooting Service Deployment
+nav_title: Troubleshooting
+menu_order: 50
+---
+
+There are several reasons why your service may fail to deploy. Some possibilities include:
+
+- Marathon isn't getting the resource offers it needs to launch the app.
+  Use the [CLI](/docs/1.9/usage/debugging/cli-debugging) or the [debugging page in the DC/OS web interface](/docs/1.9/usage/debugging/gui-debugging) to troubleshoot unmatched or unaccepted resource offers from Mesos. You can also [consult the service and task logs](/docs/1.9/administration/logging/).
```

Comment:
Thanks for the feedback, @leemunroe . @unterstein could you explain (or point me towards information on) just how offer matching works?